bower.json

```diff
@@ -37,5 +37,8 @@
   },
   "dependencies": {
     "trowel-core": "^3.3.0"
+  },
+  "resolutions": {
+    "trowel-core": "4.0.2"
```

Comment:
rather than set a resolutions, redefine the dependency "trowel-core" as "^4.0.2"